suse2013/fo/xref.xsl

```diff
@@ -44,7 +44,7 @@
                  external-destination="{$ulink.url}">
     <xsl:choose>
       <xsl:when test="count(child::node()) = 0 or
-                      (string(.) = $url) or
+                      normalize-space(string(.) = $url) or
```

Comment:
This looks a bit strange to me. I think, you want to write this:

```
 normalize-space(string(.)) = $url or
```

You can even remove the `string()` function as `normalize-space(.)` does it automatically.